CONTRIBUTING_PACKAGES.md

```diff
@@ -1,5 +1,7 @@
 # Contribute packages
 
+import CLIRemoval from '/docs/components/warnings/_cli_removal.md'
+
```

Comment:
This doc is missing the <CLIRemoval /> tag -- but I don't think the import would work in this document, regardless.  You'll have to post a regular markdown message for these, I believe.

Reply:
@Rachel-Reverie - fixed. Thanks!